faithcomesbyhearing/dbp
# It has updated dependencies to support composer version 1.10.22.

It has updated dependencies to support composer version 1.10.22.

# Description
It has updated the next dependencies to support the composer v1.10.22.

- `sentry/sentry-laravel`
- `barryvdh/laravel-ide-helper`

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-17

## How Do I QA This
The main dependency updated is the framework and we need to check the upgrade guide and check the Impact Changes.
https://laravel.com/docs/6.x/upgrade#belongs-to-update
